lib/saml/errors.rb

```diff
@@ -31,7 +35,20 @@ class UserAttributeError < SAMLError
                            message: 'User attributes contain multiple distinct CORP ID values' }.freeze,
       mhv_unverified_blocked: { code: MHV_UNVERIFIED_BLOCKED_CODE,
                                 tag: :mhv_unverified_blocked,
-                                message: 'MHV account is unverified for context requiring verified account' }.freeze
+                                message: 'MHV account is unverified for context requiring verified account' }.freeze,
```

Comment:
I'm not sure I would consider these SAML errors. Since it's technically not an issue with the SAML header on its own, but rather the comparison between SAML header and MPI response. I'd prob suggest not adding to `saml/errors` at all, and just having it encompassed in the log alone